### ISSUE TITLE
Existing Trusted Agents

### ISSUE THREAD

**Issue by dwasyl** [django-otp/django-agent-trust#4]
Is there anyway to determine if a user has existing trusted agents out there? I know you can revoke the trust for the current agent, or all agents, but to clean up a UI, I was hoping to be able to indicate if the user had any trusted devices to revoke.

Based on the fact that it's set as a cookie and not necessarily stored on the server or in the server-side session, I suspect this isn't possible but wanted to ask.

**Comment by psagers** [comment]
You're correct, agents are not automatically tracked in the database. It should be easy to do yourself, maybe with a bit of middleware.

**Comment by dwasyl** [comment]
Makes sense - logically speaking would the best place to connect be with the session? I can already tie sessions to a user, and presumably, if a user is forgetting trusted agents, the sessions associated with those agents should be invalidated as well.
